terms.html

```diff
@@ -50,46 +50,48 @@
 
   <dd>
 <a href="https://en.wikipedia.org/wiki/Public_key_infrastructure">Public key
-infrastructure</a> that does not rely on traditional
-<a href="https://en.wikipedia.org/wiki/Certificate_authority">certificate
-authorities</a> because it uses <a>decentralized identifiers</a> and
-<a>DID documents</a> to discover and verify <a>public key descriptions</a>.
+infrastructure</a> that does not rely on traditional <a
+href="https://en.wikipedia.org/wiki/Certificate_authority">certificate
+authorities</a> because it uses <a>decentralized identifiers</a> and <a>DID
+documents</a> to discover and verify <a>public key descriptions</a>.
   </dd>
 
   <dt><dfn data-lt="did controllers|did controller(s)">DID controller</dfn></dt>
 
   <dd>
-An entity that has the capability to make changes to a <a>DID document</a>.
-A <a>DID</a> might have more than one DID controller. The <a>DID controller(s)</a>
-can be denoted by the optional `controller` property at the top level of the <a>DID
-document</a>. Note that one <a>DID controller</a> might be the <a>DID subject</a>.
+An entity that has the capability to make changes to a <a>DID document</a>. A
+<a>DID</a> might have more than one DID controller. The <a>DID controller(s)</a>
+can be denoted by the optional `controller` property at the top level of the
+<a>DID document</a>. Note that a <a>DID controller</a> might be the <a>DID
+subject</a>.
   </dd>
 
   <dt><dfn>DID delegate</dfn></dt>
 
   <dd>
-An entity to whom a <a>DID controller</a> has granted permission to use
-a <a>verification method</a> associated with a <a>DID</a> via a <a>DID document</a>.
-For example, a parent who controls a child's <a>DID document</a> might permit
-the child to use their personal device in order to <a>authenticate</a>. In
-this case, the child is the <a>DID delegate</a>. The child's personal device
-would contain the private cryptographic material enabling the child to
-<a>authenticate</a> using the DID. However the child might not be permitted to
-add other personal devices without the parent's permission.
+An entity to whom a <a>DID controller</a> has granted permission to use a
+<a>verification method</a> associated with a <a>DID</a> via a <a>DID
+document</a>. For example, a parent who controls a child's <a>DID document</a>
+might permit the child to use their personal device in order to
+<a>authenticate</a>. In this case, the child is the <a>DID delegate</a>. The
+child's personal device would contain the private cryptographic material
+enabling the child to <a>authenticate</a> using the DID. However, the child
+might not be permitted to add other personal devices without the parent's
+permission.
   </dd>
 
   <dt><dfn data-lt="DID documents">DID document</dfn></dt>
 
   <dd>
 A set of data describing the <a>DID subject</a>, including mechanisms, such as
-public keys and pseudonymous biometrics, that the <a>DID subject</a> or a
```

Comment:
Agree with removing "pseudonymous biometrics" here. If we wanted to say anything about biometrics, we should have done that elsewhere, not in the terminology section.